src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -88,21 +88,23 @@ void AppHost::Initialize()
 // - Called when the app's title changes. Fires off a window message so we can
 //   update the window's title on the main thread.
 // Arguments:
+// - sender: unused
 // - newTitle: the string to use as the new window title
 // Return Value:
 // - <none>
-void AppHost::AppTitleChanged(winrt::hstring newTitle)
+void AppHost::AppTitleChanged(const winrt::Windows::Foundation::IInspectable& sender, winrt::hstring newTitle)

```

Comment:
With your new latest event proxies functional now, is it necessary/desirable to add the unused sender in as part of this event and the LastTabClosed event?  Or can/should those stay defined in their old way (just on both App and TerminalPage now)?

Reply:
Seems that I cannot get rid of this parameter. This method is attached to App::TitleChanged, which accepts a TypedEventHandler<winrt::Windows::Foundation::IInspectable, winrt::hstring>, I think the parameter should match, and TypedEventHandler needs two parameters.